class BackOffException(Exception):
    """Exception raised for recall func in wrap backoff"""
